plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/ParquetTester.java

```diff
@@ -721,7 +721,7 @@ private static void writeParquetColumnTrino(File outputFile, List<Type> types, L
             throws Exception
     {
         checkArgument(types.size() == columnNames.size() && types.size() == values.length);
-        ParquetSchemaConverter schemaConverter = new ParquetSchemaConverter(types, columnNames);
```

Comment:
We should also have some tests here with this flag disabled

Reply:
testDecimalBackedByINT32 and testDecimalBackedByINT64 already take care of testing write + read for integer encoded decimals